go.mod

```diff
@@ -50,3 +50,98 @@ require (
 	k8s.io/api v0.19.6
 	k8s.io/apimachinery v0.19.6
 )
+
```

Comment:
whats with the 2 require sections?

Reply:
https://golang.org/doc/go1.17#go-command